### PR TITLE
:space_invader: first pass at an about page

### PR DESCRIPTION
**Pull request by mgiraldo**
you can better see/read the text in this link: https://github.com/NYPL/design-toolkit/blob/about/docs/_sections/about.md

### CODE REVIEW

**Comment by ablwr** [comment]
LGTM but I defer copy tweaking to the copy-masters.

File: docs/_sections/about.md [hunk 1]
```diff
@@ -3,22 +3,45 @@ title: About This Toolkit
...
 ---
 
-### NYPL Web Presence As A Public Space
+### Introduction
 
-Coming soon
+This toolkit aims to serve multiple purposes:
 
-### Modularity
+- facilitate the creation of new web user interfaces
```
Comment (wlla) [inline docs/_sections/about.md]: update to: "facilitate the rapid creation of new web user interfaces" ?

File: docs/_sections/about.md [hunk 2]
```diff
@@ -3,22 +3,45 @@ title: About This Toolkit
...
+- ensure consistency in design and quality
+- serve as the main repository for documenting the web design efforts at NYPL Digital
```
Comment (wlla) [inline docs/_sections/about.md]: Add a 4th bullet point?
"- establish and promote universal design best practices"

File: docs/_sections/about.md [hunk 3]
```diff
@@ -3,22 +3,45 @@ title: About This Toolkit
...
+- they are already installed in your computer or mobile device so there is no additional download required to use them, thus reducing bandwidth usage costs
+- having the nuances of **bold text**, _italic text_, or `code text` also increases the size of the download (and we want those)
+- our materials are multilanguage, making finding a custom font that supports, say, Persian or Chinese very expensive
+- even if a custom font supports every single language, like the commendable [Noto free font](https://www.google.com/get/noto/) by Google, it will be a _very_ big download for our patrons (the complete Google Noto set is 970 megabytes!)
+
+These are just the main reasons why we decided, for now, to use system fonts in this toolkit.
+
+### Accessibility And Mobile First
+
+This toolkit is designed with accessibility and mobile use in mind first. All features should be available to all users in all devices. If you see something that does not conform to this goal, please [open an issue in Github](https://github.com/NYPL/design-toolkit/issues/new).
```
Comment (wlla) [inline docs/_sections/about.md]: "This toolkit puts universal design and mobile use first. All features should be accessible to all users in all devices. We strive to conform to WCAG 2.0 AA standard. If you see something that does not conform to this goal and standard, please open an issue in Github."

Comment (mgiraldo) [inline docs/_sections/about.md]: fixed

File: docs/_sections/about.md (hunk 2 above)
Comment (mgiraldo) [inline docs/_sections/about.md]: fixed

**Pull request merged**